sdk/src/modules/markets/markets.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { GmxSdk } from "../../index";
+import { arbitrumSdk, arbitrumSdkConfig } from "utils/testUtil";
+
+describe("Markets", () => {
+  describe("getMarkets", () => {
+    it("should be able to get markets", async () => {
+      const response = await arbitrumSdk.markets.getMarkets();
+      expect(response).toBeDefined();
+    });
+  });
+
+  describe("getMarkets", () => {
+    it("should be able to get markets data", async () => {
+      const marketsData = await arbitrumSdk.markets.getMarkets();
+      expect(marketsData.marketsAddresses).toBeDefined();
+      expect(marketsData.marketsData).toBeDefined();
+    });
+
+    it("should respect config filters", async () => {
+      const sdk = new GmxSdk({
+        ...arbitrumSdkConfig,
+        markets: {
+          "0x47c031236e19d024b42f8AE6780E44A573170703": {
+            isListed: false,
+          },
+        },
+      });
+      const baseSdkResponse = await arbitrumSdk.markets.getMarkets();
+      const sdkResponse = await sdk.markets.getMarkets();
+      expect(baseSdkResponse.marketsData?.["0x47c031236e19d024b42f8AE6780E44A573170703"]).toBeDefined();
+      expect(sdkResponse.marketsData?.["0x47c031236e19d024b42f8AE6780E44A573170703"]).not.toBeDefined();
+    });
+  });
+
+  describe("getMarketsInfo", () => {
+    it("should be able to get markets info", async () => {
+      const response = await arbitrumSdk.markets.getMarketsInfo();
+      expect(response).toBeDefined();
+    });
+  });
+
+  describe("getDailyVolumes", () => {
+    it("should be able to get daily volumes", async () => {
+      const response = await arbitrumSdk.markets.getDailyVolumes();
+      expect(response).toBeDefined();
+    }, 30_000);
+  });
+});
```

Comment:
can you add a test to check that all markets listed in sdk config are same as in frontend? otherwise it's hard to visually validate each of them, would also simplify publishing new markets

Reply:
It's not 100% possible as markets in SDK are fetched from chain and filtered by `/markets` endpoint

Reply:
maybe not in tests then,
you build markets map in `src/config/static/markets.ts`, I guess you can throw some error there in case market for any chain is not listed in `src/config/markets` (I might confused some paths you should get the idea)